articles/native-platforms/android/07-authorization.md

```diff
@@ -0,0 +1,64 @@
+ ---
+ title: Auth0 Android Quickstarts - Authorization
+ description: This tutorial will show you how to use the Auth0 authentication API in your Android project to create a custom login screen.
+ seo_alias: android
+ ---
+ 
+ ## Android - Authorization Tutorial
+ 
+ This is a simple quickstart that will show you how to use Auth0 to create access roles for your users. This way you can authorize or deny content to different users.
+ 
+ ::: panel-info System Requirements
+ This tutorial and seed project have been tested with the following:
+ 
+ * AndroidStudio 2.0
+ * Emulator - Nexus5X - Android 6.0 
+   :::
+ 
+ 
+ ### Before Starting
+ 
+ Be sure that you have completed the [User Profile](04-user-profile.md) Quickstart.
+ 
+ ### 1. Create a Rule to assing roles
+ 
+ First, you need to create a rule that assigns your users either an `admin` role, or a single `user` role. To do so, go to the [new rule page](https://manage.auth0.com/#/rules/new) and select the "*Set Roles To A User*" template, under *Access Control*. Then, replace this line from the default script:
+ 
+  
+   ``` 
+   if (user.email.indexOf('@admin.com') > -1)
```

Comment:
Please, make sure that by default it says `@admin.com`. Check it out in the [new rule page](https://manage.auth0.com/#/rules/create). In my case, it defaults to `@gonzo.com`, although @ntotten said that it will be `@example.com` in the future. I ended up writing `example.com` as default in the iOS-Swift tutorial.